src/player.h

```diff
@@ -167,6 +181,7 @@ class Player
 	v3f m_speed;
 
 	std::vector<HudElement *> hud;
+	PlayerSettings m_settings;
```

Comment:
`m_settings` is looks like it should be a settings class, I suggest renaming to `m_player_settings` or similar.

Reply:
Ok, will do!

Reply:
Changed.